storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -147,6 +149,12 @@ static bool reopen_log_files(lsn_t desired_lsn) {
       srv_read_only_mode, log.m_encryption_metadata, files, format,
       creator_name, log_flags, log_uuid);
 
+  if (desired_lsn == 0) {
+    if (res != Log_files_find_result::FOUND_VALID_FILES)
+      return false;
+    else
+      return true;
+  }
```

Comment:
One caller of this functions prints the desired_lsn along with it. The other one doesn't. May be it should be printed by this function instead of callers.

Please check callers of reopen_log_files()

Reply:
@rahulmalik87 did you address this?